src/test/java/com/google/api/generator/engine/writer/ImportWriterVisitorTest.java

```diff
@@ -778,6 +780,21 @@ public void writeReferenceConstructorExprImports_withArgs() {
     referenceConstructorExpr.accept(writerVisitor);
   }
 
+  @Test
+  public void writeArithmeticOperationExprImports() {
+    MethodInvocationExpr lhsExpr =
+        MethodInvocationExpr.builder()
+            .setStaticReferenceType(TypeNode.withReference(ConcreteReference.withClazz(Expr.class)))
+            .setMethodName("getSomething")
+            .setReturnType(TypeNode.STRING)
+            .build();
+    ValueExpr rhsExpr = ValueExpr.withValue(NullObjectValue.create());
```

Comment:
Let's make this an expression that requires an import (and `null` is already tested elsewhere).

Reply:
In java, `String c = "ac" + null;`, it is worthy to add this test case here.
sg, I will add a test case that requires an import.